yb-voyager/cmd/import.go

```diff
@@ -429,3 +440,28 @@ func validateTruncateTablesFlag() {
 		utils.ErrExit("Error --truncate-tables true can only be specified along with --start-clean true")
 	}
 }
+
+func validateEnableFastPathFlag() {
+	if !enableFastPath && onPrimaryKeyConflictAction != "" {
+		utils.ErrExit("Error: To use --on-primary-key-conflict, you must also enable --enable-fast-path.")
```

Comment:
hmm, this doesn't seem right. let's think about this. 
on-primary-key-conflict is a behavior decision. 
enable-fast-path is a performance optimization. 

I don't see why they have to be dependent on each other.

Reply:
We discussed this right.
We dont want to assume have behaviour for retries in case of fast path.
So user should voluntarily inform us what they are ok with.

Reply:
Yeah, sorry I didn't communicate this properly. Internally for us, yes they are dependent. But from user's perspective, I'm not so sure if the combination of flags are simple enough to understand. 
Haven't thought about it completely, but maybe we only need "on-primary-key-conflict"?